### PR TITLE
camera usage tweaks [CR]

### PR DESCRIPTION
**Pull request by sparr**

### CODE REVIEW

**Comment by Lain-** [comment]
Nice! I like the "selfie" part!

File: src/iuse.cpp [hunk 1]
```diff
@@ -8445,8 +8445,14 @@ int iuse::camera(player *p, item *it, bool, point)
         }
 
         if (aim_point.x == p->posx() && aim_point.y == p->posy()) {
-            p->add_msg_if_player(_("You decide not to flash yourself."));
-            return 0;
+            if (query_yn(_("Are you sure you want to flash yourself?"))) {
+                p->add_msg_if_player(_("Great selfie. The flash blinds you."));
+                p->add_effect("blind", rng(5, 10));
```
Comment (KA101) [inline src/iuse.cpp]: But it's sunny out (no need for flash!)/I've got sunglasses on/I've got antiglare gear on/etc!

More to the point, I can understand the use of a short-term self-blinder, but frankly we'd be better off with a blindfold (if one hasn't yet been made) rather than making people pull nethack stunts.  :-/  Sorry.

Comment (sparr) [inline src/iuse.cpp]: @KA101 to be honest, this was at least 50% inspired by me needing a way to blind myself for testing on my other currently PR'd branch. Should I re-submit without this change?

Comment (KA101) [inline src/iuse.cpp]: Either pull the change from this one, or add a check for welding/RM13/bionic glare-protection, please.  Should just be another commit or two, and push 'em to this branch: that'll auto-update the PR.

**Comment by sparr** [comment]
@KA101 removed the selfie part

**Comment by KA101** [comment]
I think this needs adjusted to accommodate the blindness PR landing.  If you can, please do; otherwise, we can but may be a bit longer at it.

**Comment by sparr** [comment]
@KA101 sure, I can probably re-do some of these changes to accommodate the new blindness thing you put in for the blindfold.

**Comment by sparr** [comment]
@KA101 ok, merged master containing the blindness patches. I think this is ready?

**Comment by KA101** [comment]
OK, thanks.  I'll see about rolling it into the necropolis merge (building that now).
